service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/NarrativeStatementMapperTest.java

```diff
@@ -41,6 +46,16 @@ public void setUp() {
         narrativeStatementMapper = new NarrativeStatementMapper(randomIdGeneratorService);
     }
 
+    @BeforeAll
+    public static void initialize() {
+        TimeZone.setDefault(TimeZone.getTimeZone(ZoneOffset.UTC));
+    }
+
+    @AfterAll
+    public static void deinitialize() {
+        TimeZone.setDefault(null);
+    }
+
```

Comment:
Should we have any tests classes that use time inherit these from a super?

Reply:
This could be done as a part of this [task](https://gpitbjss.atlassian.net/browse/NIAD-1060).